apps/derive/elpi/invert.elpi

```diff
@@ -65,18 +66,20 @@ trivial-arity (arity (prod Name Ty In)) Ni (parameter ID explicit Ty Out) :-
   ID is "idx" ^ {term_to_string Ni},
   Ni1 is Ni + 1,
   @pi-decl Name Ty p\ trivial-arity (arity (In p)) Ni1 (Out p).
-trivial-arity (arity T) _ (arity T).
+trivial-arity (arity T) _ (arity T1) :-
+  % refresh univs
+  (pi u\ copy (sort (typ _)) (sort (typ u))) => copy T T1.
```

Comment:
Hum this looks like a little bug, also this refresh should be part of the standard API